test/e2e/integration/i18n.e2e.ts

```diff
@@ -241,4 +241,39 @@ describe('I18N', () => {
     changeCurrentLocaleToDe();
     cy.get("#i18n-img").should("have.attr", "src", de.imgPath);
   });
+
+  describe("with custom elements", () => {
+    it("can pass interpolated translations to custom elements bindables", () => {
```

Comment:
Isn't it better to have the tests generated looping over a data array? I mean that pattern is already used before.

Reply:
I was thinking about it first but then decided against it since there are only a few tests, which are again sub-grouped and it appeared to me the readability might be better writing them out individually. I'm happy to change that though if you feel it should be consistent.

Reply:
I am with you. That's why I opted for this format before. I really don't mind the inconsistency. For vNext, I think I will go with this one, rather than looping over an array.